packages/core/spec/browser/asciidoctor-spec.js

```diff
@@ -43,6 +45,28 @@
         const html = asciidoctor.convert('include::' + testOptions.baseDir + '/spec/fixtures/include.adoc[]', opts)
         expect(html).to.include('include content')
       })
+
+      it('should resolve a stylesheet using chrome:// protocol (root)', function () {
+        const attributes = {
+          'stylesheet': 'asciidoctor-plus.css',
+          'stylesdir': 'chrome://asciidoctor/extension/css',
+          'copycss': false
+        }
+        const options = {
+          doctype: 'article',
+          safe: 'unsafe',
+          standalone: true,
+          attributes
+        }
+        try {
+          asciidoctor.convert('Hello world', options)
+          expect.fail('the resource does not exist and the converter should throw an exception!')
+        } catch (e) {
+          // the resource does not exist but chrome:// is a root path and should not be prepended by "./"
+          // we expect the processor to try to load the resource 'chrome://asciidoctor/extension/css/asciidoctor-plus.css' and fail!
+          expect(e.message).to.include(`Failed to load 'chrome://asciidoctor/extension/css/asciidoctor-plus.css'`)
```

Comment:
@bodtx I've added comments to explain this test.
Basically I've copied and adapted your code from [chrome/content/myCompose.js](https://framagit.org/bodtx/asciidoctor_tb/blob/f4f0d83d92c5372c6af4521bdaf0d8cce9630ecf/chrome/content/myCompose.js#L71-85) but since the resource `chrome://asciidoctor/extension/css/asciidoctor-plus.css` does not exist, I'm catching the exception and asserting that the resource path was not prepended by `./`. 

Does it make sense?

Reply:
indeed the `./` added to my url was the problem.
if there is just `chrome://` the url should be ok

Reply:
Thanks for your feedback!